src/icrar/leap-accelerate/model/casa/MetaData.cc

```diff
@@ -182,6 +184,30 @@ namespace casalib
         dd3d(2,2) = std::cos(dlm_dec);
     }
 
+    void MetaData::SetDD(const icrar::MVDirection& direction)
+    {
+        if(!dd.is_initialized())
+        {
+            dd.reset(casacore::Matrix<double>(3,3));
+        }
+
+        auto& dd3d = dd.value();
+        dlm_ra = direction(0) - phase_centre_ra_rad;
+        dlm_dec = direction(1) - phase_centre_dec_rad;
+
+        dd3d(0,0) = cos(dlm_ra) * cos(dlm_dec);
+        dd3d(0,1) = -sin(dlm_ra);
+        dd3d(0,2) = cos(dlm_ra) * sin(dlm_dec);
+        
+        dd3d(1,0) = sin(dlm_ra) * cos(dlm_dec);
+        dd3d(1,1) = cos(dlm_ra);
+        dd3d(1,2) = sin(dlm_ra) * sin(dlm_dec);
+
+        dd3d(2,0) = -sin(dlm_dec);
+        dd3d(2,1) = 0;
+        dd3d(2,2) = cos(dlm_dec);
+    }
+
```

Comment:
Suspiciously familiar again, I'm sure it's taken care of in a different PR or was a wrong merge.

Reply:
Ahh, this I quickly added in as a duplicate of the above code for testing and was meaning to take out. I'll replace it with this:
```
    void MetaData::SetDD(const icrar::MVDirection& direction)
    {
        SetDD(ConvertDirection(direction));
    }
```